op-guide/docker.md

```diff
@@ -1,9 +1,196 @@
-# Docker
+# Run a TiDB Cluster in Docker on a Single Host
 
-## Run with `docker-compose`
+## Preparation
 
-A simple `docker-compose.yml`:
+Before you start, make sure that:
+
++ Installed the latest version of [Docker](https://www.docker.com/products/docker) 
++ Pulled the TiDB/TiKV/PD docker images from PingCAP's Docker Hub repositories
```

Comment:
This section should be in the same format as same section in "docker-across-multiple-hosts.md".